fr.rubet.rpn.yml

```diff
@@ -0,0 +1,16 @@
+app-id: fr.rubet.rpn
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: rpn
+modules:
+  - mpfr.yaml
```

Comment:
this does not need to be in a separate yml file.

Reply:
done